sci-libs/beagle/beagle-3.0.3-r1.ebuild

```diff
@@ -1,11 +1,11 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
 DESCRIPTION="Open BEAGLE, a versatile EC/GA/GP framework"
 SRC_URI="mirror://sourceforge/beagle/${P}.tar.gz"
```

Comment:
So Homepage is now on Github. What about the releases? Still only on Sourceforge?

Reply:
The 3.0.3 release is still hosted on SourceForge, although another exists on the Google Code Archive (https://code.google.com/archive/p/beagle/downloads). I didn't change the SRC_URI because the current one still works.